app/Libraries/BitbucketNotification.php

```diff
@@ -25,6 +25,15 @@ public function __construct($notification)
 
     public function run()
     {
+        $action = $this->notification->header("X-Event-Key");
+        $validActions = ["pullrequest:created"];
+
+        // Only 'pullrequest:created' action is supported at this moment
+        if (!in_array($action, $validActions)) {
+            echo "The action '{$action}' is not valid. Only 'pullrequest:created' action is supported at this moment.\n";
```

Comment:
Hey @DannyFeliz check this

Reply:
Check this you too @dawin09

Reply:
Check this @gabbanaesteban

Reply:
Check this @dawin09 and give me a feedback

Reply:
Check this @dawin09 and give me a feedback